### DISCUSSION TITLE
Handle iOS Push Notification in background when app is closed

### DISCUSSION THREAD

**Discussion by game8149** [invertase/react-native-firebase#4291] [Q&A] [answered]
Hi, 
I have a question respect to how iOs handle their notifications when an app is closed. There's some idea? In Android, these notifications are handled even app is closed, but on iOS they aren't handled.

**Comment by mikehardy** [comment, marked as answer]
There are so many issues on this, and so many questions. Please search for them, and follow their advice.
This answer is very direct yes, I don't intend to be rude but people really struggle with this, so as you read the other issues please understand the module works. If you have a problem, it is with your project. I say that in advance so that you continue to focus troubleshooting when you hit problems (which you will) on your project, as the fixes will be necessary in your project config and API usage, not in the module
